views/lti/deliverableNavbar.jade

```diff
@@ -26,6 +33,22 @@ script
       navItems.forEach(initializeNavItemEvents)
       navSubTypes.forEach(initializeNavSubTypeEvents)
 
+      const deliverableDropdown = document.querySelector('.deliverable-navbar-header')
```

Comment:
This will only fetch one DOM element with this class? Is this on purpose? Also, would it be better to use id instead of class if we know we'll only have one element?